modules/release-notes/partials/docs-server-7.2.4-release-note.adoc

```diff
@@ -0,0 +1,104 @@
+
+[#release-724]
+== Release 7.2.4 (January 2024)
+
+Couchbase Server 7.2.4 was released in November 2023.
+This maintenance release contains new features and fixes to issues.
+
+[#new-features]
+=== New Features and Enhancements
+
+* REST interface added to set the retention period of audit files
+* Erlang upgraded to version{nbsp}25.
+* Added `Memcached` alert.
+
+
+=== Cluster Manager
+
+[#table-new-features-724-cluster-manager, cols="10,40"]
+|===
+|Feature | Description
+
+| https://issues.couchbase.com/browse/MB-59503/[MB-59503]
+| The request is for `ns_server` to provide a REST interface to enter the minimum duration (in minutes) that previously rotated audit files will be retained. This value will be converted to seconds and passed through to memcached. The audit daemon, run during the log rotation task, will remove any files created more than X seconds ago.
+
+|===
+
+=== Data Service
+
+[#table-new-features-724-data-service, cols="10,40"]
+|===
+|Feature | Description
+
+| https://issues.couchbase.com/browse/MB-8925/[MB-58925]
+| Couchbase Server 7.2.2 currently uses Erlang{nbsp}24. As Couchbase Server 7.2 will be maintained until 2026, Erlang has been upgraded to version{nbsp}25 to ensure we receive future code fixes.
+
+
+|  https://issues.couchbase.com/browse/MB-59261/[MB-59261]
+| `Memcached` alert added. +
+Add an alert when the number of memcached connections has exceeded 90% of the limit.
+|===
+
+
+== Fixed Issues
+
+=== Cluster Manager
+
+[#table-known-issues-724-cluster-manager, cols="10,40,40"]
+|===
+|Issue | Description | Resolution
+
+| https://issues.couchbase.com/browse/MB-59237/[MB-59237]
+| Record OS disk I/O metrics into Prometheus. Currently, there is not enough information recorded in the logs to diagnose slowdowns in the system related to Disk I/O
```

Comment:
@RayOffiah I've added a comment to this JIRA asking the engineer to provide a brief description of the problem and resolution. It's not clear to me whether all of this was actually implemented.

Reply:
Okay, I'll chase that one up.